CONTRIBUTING.md

```diff
@@ -53,10 +53,10 @@ cd jquery-ui
 git remote add upstream git://github.com/jquery/jquery-ui.git
 ```
 
-* Get in the habit of pulling in the "upstream" master to stay up to date as jQuery UI receives new commits.
+* Get in the habit of pulling in the "upstream" main to stay up to date as jQuery UI receives new commits.
```

Comment:
What about the addition "branch" we talked about? See https://jqueryui-dev.slack.com/archives/GPFN73HHP/p1612705325000200?thread_ts=1612544137.001000&cid=GPFN73HHP

Reply:
Ha! I added that word to the commit message but not here. I'll do that.

Reply:
@fnagel PR updated, please have a look.

Reply:
@mgol Perfect!